src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -1938,7 +1944,7 @@ namespace Akka.Actor.Internal
     public abstract class AbstractStash : Akka.Actor.IStash
     {
         protected AbstractStash(Akka.Actor.IActorContext context) { }
-        public int Capacity { get; }
+        public virtual int Capacity { get; }
```

Comment:
`AbstractStash.Capacity` is `virtual` so it can be overridden and made constant inside `UnboundedStashImpl`.